api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java

```diff
@@ -725,7 +725,7 @@ public CohortDefinition getPatientsWithStartOrContinueOnRapidFlow() {
     cd.addParameter(new Parameter("onOrBefore", "Before Date", Date.class));
     cd.addParameter(new Parameter("location", "Location", Location.class));
     cd.setQuery(
-        CommonQueries.getLastCodedObsBeforeDate(
```

Comment:
Maybe you need to remove duplicate method in CommonQueries.java class, that seem to have transfereed to another class

Reply:
This already handled

Reply:
Thanks for this one @ningosi